crypto/poseidon/poseidon.go

```diff
@@ -175,3 +175,8 @@ func HashFixed(inpBI []*big.Int) (*big.Int, error) {
 	rE.ToBigIntRegular(r)
 	return r, nil
 }
+
+// Deprecated HashFixed entry, with domain field is 0
+func HashFixed(inpBI []*big.Int) (*big.Int, error) {
```

Comment:
print out a warning message that this function is deprecated.

Reply:
done